BitmovinComscoreAnalytics/Classes/ComScoreBitmovinAdapter.swift

```diff
@@ -80,10 +80,14 @@ extension ComScoreBitmovinAdapter: PlayerListener {
     }
 
     func onPaused(_ event: PausedEvent) {
+        
+        //TODO: remove once we have tvOS support for this method
+        #if(iOS)
```

Comment:
Does this even work? I assume this macro check is not working.

It should be `#if os(iOS)`